Source/Controllers/MainViewControllers/TableStructure/SPTableStructure.m

```diff
@@ -1447,9 +1448,20 @@ - (void)loadTable:(NSString *)aTable
 			}
 
 			// MySQL < 4.1 does not support collations (they are part of the charset), it will be nil there
-
-			[theField setObject:encoding forKey:@"encodingName"];
-			[theField setObject:collation forKey:@"collationName"];
+            if(collation != nil){
```

Comment:
Do we really need this if docu says mySQL 4.1? 😂

Reply:
think the docs are wrong... this threw on 5.7 I think.